app/models/cart.rb

```diff
@@ -0,0 +1,3 @@
+class Cart < ApplicationRecord
```

Comment:
Ignore this comment. I will update the config so it doesn't complain about this.